doc/lsp-status.txt

```diff
@@ -103,7 +103,8 @@ config({config})                                   *lsp-status.config()*
                   accepted as the symbol currently containing the cursor.• 
                 • `current_function`: Boolean, `true` if the current function
```

Comment:
What do you think of changing this (here, in the API, and in the other docs) to be called `current_scope` or similar? The old name seems no longer accurate.

Reply:
This makes sense to me, but it's also a breaking change. Not sure whether the readability is more important.

Reply:
Perhaps we merge this as-is, then announce a breaking change and make it in a separate PR?

Reply:
But I would merge this, we can discuss naming in another PR - it seems like an adjacent but not identical issue that needs some further discussion.

Reply:
I agree.